src/Styleguide/Artwork/__stories__/Banner.story.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import { storiesOf } from "@storybook/react"
+import { storiesOf } from "storybook/storiesOf"
```

Comment:
@alloy - your suggestion worked great. What I was a bit unsure of was where to put it. As more stuff comes online i figured that we would be using it more so I stashed it in `src/__stories__` next to the config file and created this alias, but if it makes more sense just to keep it inside of `Styleguide` i can move it out.

Reply:
Nice 😎